File: Developer/CocoaTouchLayer/UIKit/YJViewGeometry/YJViewGeometry/ViewController.m
//
//  ViewController.m
//  YJViewGeometry
//
//  Created by admin on 16/5/31.
//  Copyright © 2016年 YJ. All rights reserved.
//

#import "ViewController.h"
#import "UIView+YJTViewGeometry.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    UIView *view = [[UIView alloc] initWithFrame:CGRectMake(50, 20, 100, 100)];
    view.backgroundColor = [UIColor redColor];
    [self.view addSubview:view];
    NSLog(@"%@", NSStringFromCGPoint(view.originFrameInWindow));
    view.centerXFrame = 100;
    NSLog(@"%@", NSStringFromCGRect(view.frame));
    
    self.view.topBounds = -130;
}

@end

File: Demo/Swift/AppFrameworks/UIKit/YJViewGeometry/YJViewGeometry/ViewController.swift
//
//  ViewController.swift
//  YJViewGeometry
//
//  Created by 阳君 on 2019/5/23.
//  Copyright © 2019 YJCocoa. All rights reserved.
//

import UIKit
import YJCocoa

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        let view = UIView.init(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
        view.frameTop = 100
        view.backgroundColor = UIColor.red
        self.view.addSubview(view)
    }

}

